frontend/src/components/Applications/Applications.tsx

```diff
@@ -13,27 +20,32 @@ export default function Applications() {
     const [reloadRotate, setReloadRotate] = useState<boolean>(false);
     const [addRotate, setAddRotate] = useState<boolean>(false);
     const [showForm, setShowForm] = useState<boolean>(false);
+    const [selectedApplication, setSelectedApplication] = useState<Application | null>(null);
 
-    function fetchApplications() {
+    async function fetchApplications() {
```

Comment:
At this moment i need the (complicated) version with the loading State, because i want to show the loading spinner for a while. The data fetching ist too fast and i don't like the "aufblitzen" - it makes me nervous... ;-)